.github/workflows/tests.yml

```diff
@@ -22,6 +22,7 @@ jobs:
             "@liveblocks/react-lexical",
             "@liveblocks/react-tiptap",
             "@liveblocks/node-lexical",
+            "@liveblocks/node-prosemirror",
```

Comment:
I took the liberty of publishing an empty 0.0.1 for this package here: https://www.npmjs.com/package/@liveblocks/node-prosemirror. Now the CI publish step should Just Work™.

Reply:
and that's why you're my favorite person 🙇